### ISSUE TITLE
Single Mode required?

### ISSUE THREAD

**Issue by m-mohr** [stac-extensions/stereo-imagery#3]
The "single mode" (one item with multiple assets) has some drawbacks:
- Geometry, view angles and time usually (slightly) different
- View Extension not defined for assets

Both Planet and Up42 would use the "multi mode". The "single mode" was proposed by @matthewhanson, but we don't have any examples where this mode would be "superior" right now. 

Thus as a first step I recommend the multi mode in the docs, but the next question is: Do we really need both modes?

cc @cholmes @PelJon

**Comment by cholmes** [comment]
I definitely lean towards _not_ including something that's an idea with no specific real example. Like I think it's a good idea, happy to add it when someone needs it. Maybe can make an issue for it and document the original idea, and then we can do a 1.1 if someone has a clear need for it.

**Comment by PelJon** [comment]
I would agree with @cholmes here. Keep it lean and expand with a concrete use case in mind.

**Comment by m-mohr** [comment]
Waiting on @matthewhanson to chime in as he was originally proposing the "single mode".

**Comment by matthewhanson** [comment]
I had originally proposed the single Item view because I was thinking of a stereo image pair as a a set of imagery that was collected together and that would be used together.  Even if one were not to use them as a traditional stereo pair (to determine heights) and only as single images, having them as two different items are for most practical purposes, a duplicate.  They are an image of the same area at the same time, just with different view angles.

For stereo imagery which may have been taken at different times, without the express purpose of using as a stereo image pair, then separate items definitely make sense.

However, using different Items is certainly fine in all cases and if no one is representing a pair as a single Item then I see little reason to include it in the extension.

**Comment by matthewhanson** [comment]
@m-mohr I am wondering about your point that the View extension isn't defined for assets.  It defines properties, which can be defined at the Item or Asset level.

**Comment by m-mohr** [comment]
@matthewhanson I just looked at the Readme, which only said "Item Properties", which in the past meant only allowed in Item Properties and not assets, which would have had implications on the schema. But I just looked at it closer in the schema and there we already made the switch to the newer extension schema. So in theory it works in assets, but is not reflected in the README yet. I just updated it accordingly: https://github.com/stac-extensions/view/commit/9f713671340ca70f4c3bbe13be1463f79467be46
So my point above is not valid. The view properties can be used in assets.

What I'll do is to remove the "Single Mode" in a way that we can add it later again without breaking the extension. And then I'll already create a PR of changes to add it again in case we may need it. That keeps it simple for now, but allows to easily add in case someone shows up with an actual example of the Single Mode.
